go-controller/pkg/ovn/address_set/fake_address_set.go

```diff
@@ -272,6 +273,28 @@ func (as *fakeAddressSets) AddIPs(ips []net.IP) error {
 	return nil
 }
 
+// TEMP
+func (as *fakeAddressSets) AddIPsReturnsOps(ips []net.IP) ([]ovsdb.Operation, error) {
```

Comment:
@tssurya you could have AddIPs() call this function and then run `TransactAndCheck()` on the returned ops